### PR TITLE
Update p3 docs

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR updates the documentation to match recent UI changes.

### CODE REVIEW

File: docs/photoniq/p3/p3-optimizations.md [hunk 1]
```diff
@@ -41,6 +41,10 @@ Strips superfluous characters from HTML files, decreasing size. Results in quick
 
 Eliminates unnecessary characters from JavaScript files, reducing size. Enhances network transmission speed and execution times, increasing page speed.
 
+## Move Script Tags From Top to Bottom
```
Comment (diana-macrometa) [inline docs/photoniq/p3/p3-optimizations.md]: All blank optimizations on this page need definitions.

File: docs/photoniq/p3/view-p3-optimizations.md [hunk 2]
```diff
@@ -5,13 +5,30 @@ title: View P3 Optimizations
...
+
+## Optimization Fields
+
+- **Policy Name** - Name of the policy setting the optimizations.
+- **URL Pattern** - URL pattern associated with the policy.
+- **URL** - Specific URL being optimized.
+- **Progress** - Shows where in the optimization process the optimization is.
+  - **A** - Analysis. P3 is analyzing the page.
+  - **V** - Validation. P3 is validating the page. For more information about validation, refer to [View P3 Validations](view-p3-validations.md).
+  - **R** -
```
Comment (diana-macrometa) [inline docs/photoniq/p3/view-p3-optimizations.md]: Need definition for R.

File: docs/photoniq/p3/view-p3-validations.md [hunk 3]
```diff
@@ -13,16 +13,27 @@ Perform the following steps to view P3 validations:
...
+## Validations Table Fields
+
+The Validations table provides the following fields:
+
+- **Origin** - The base URL that was validated.
+- **URL Pattern** - The specific URL that was validated.
+- **Last Updated** - When the optimization was last updated.
+- **Validation** - Types of validation: **Visual** and **Functional**. Click one to view validation results.
+
 ## Visual Validation
```
Comment (diana-macrometa) [inline docs/photoniq/p3/view-p3-validations.md:33]: No content changes in this section, just updated the order.

File: docs/photoniq/p3/p3-api-keys.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 90
+title: Manage P3 API Keys
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This page provides information to help you manage your P3 API keys.
+
```
Comment (diana-macrometa) [inline docs/photoniq/p3/p3-api-keys.md:9]: @dwivedianurag and @nobuyoshi-macrometa - When I created API keys, they did not display for me. Is that intentional? Or will there be a way to view a list of API keys in the UI?

Comment (dwivedianurag) [reply docs/photoniq/p3/p3-api-keys.md:9]: Currently there is no way to view list of apikeys it is intentional for now @diana-macrometa

File: docs/photoniq/p3/p3-optimizations.md [hunk 5]
```diff
@@ -13,6 +13,10 @@ Bundles multiple CSS files into one, reducing HTTP requests. This accelerates pa
 
 Consolidates multiple JavaScript files into a single file. Reduces HTTP requests, enhancing page load efficiency by streamlining file downloading and parsing. This process not only reduces the overhead associated with each HTTP request but also allows for better compression ratios, as similar code across files can be more efficiently compressed.
 
+## Compression for Mobile Devices
```
Comment (dwivedianurag) [inline docs/photoniq/p3/p3-optimizations.md]: Optimiziations for slow network
This should be the titile @diana-macrometa

File: docs/photoniq/p3/view-p3-audit-log.md [hunk 6]
```diff
@@ -21,7 +21,7 @@ Perform these steps to view the P3 audit log:
 
 The following fields list information about the audit log lines:
 
+- **Policy Name** - The name of the policy that the logged status applies to.
```
Comment (dwivedianurag) [inline docs/photoniq/p3/view-p3-audit-log.md:24]: It is the name of the policy which the user is going to create

**Review by dwivedianurag — APPROVED** [review]
Ok

File: docs/photoniq/p3/view-p3-audit-log.md (hunk 6 above)
Comment (diana-macrometa) [reply docs/photoniq/p3/view-p3-audit-log.md:24]: I am confused. The audit log can only log things that have already been created, so the policy needs to have been created at this point.

**Pull request merged**